The job **`mingw-check`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/14632137908/job/41056289518) [(plain)](https://github.com/rust-lang/rust/commit/3a14991e77b890c6971e9d02e83d25bc281f2a70/checks/41056289518/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
    = note: `-D rustdoc::unescaped-backticks` implied by `-D warnings`
    = help: to override `-D warnings` add `#[allow(rustdoc::unescaped_backticks)]`
help: the opening backtick of a previous inline code may be missing
    |
263 | //! to the contents of the [`Result`] to produce a boolean value. If the `[`Result`]` does not have the expected variant
    |                                                                          +
help: if you meant to use a literal backtick, escape it
    |
264 | //! then `false\` is returned instead without executing the function.
    |                +

error: could not document `core`
warning: build failed, waiting for other jobs to finish...
[RUSTC-TIMING] core test:false 12.779
Command has failed. Rerun with -v to see more details.
Build completed unsuccessfully in 0:00:13
  local time: Thu Apr 24 02:58:01 UTC 2025
  network time: Thu, 24 Apr 2025 02:58:01 GMT
##[error]Process completed with exit code 1.
Post job cleanup.
```

</details>

--- a/library/core/src/result.rs
+++ b/library/core/src/result.rs
@@ -260,8 +260,8 @@
 //! is [`Ok`] or [`Err`], respectively.
 //!
 //! The [`is_ok_and`] and [`is_err_and`] methods apply the provided function
-//! to the contents of the [`Result`] to produce a boolean value. If the [`Result`]` does not have the expected variant
-//! then `false` is returned instead without executing the function.
+//! to the contents of the [`Result`] to produce a boolean value. If the [`Result`] does not have the expected variant
+//! then [`false`] is returned instead without executing the function.
 //!
 //! [`is_err`]: Result::is_err
 //! [`is_ok`]: Result::is_ok

--- a/library/core/src/result.rs
+++ b/library/core/src/result.rs
@@ -342,7 +342,7 @@
 //! [`Some(v)`]: Option::Some
 //! [`transpose`]: Result::transpose
 //!
-//! This method transforms the contained value of the [`Ok`] variant:
+//! These methods transform the contained value of the [`Ok`] variant:
 //!
 //! * [`map`] transforms [`Result<T, E>`] into [`Result<U, E>`] by applying
 //!   the provided function to the contained value of [`Ok`] and leaving
@@ -354,7 +354,7 @@
 //! [`map`]: Result::map
 //! [`inspect`]: Result::inspect
 //!
-//! This method transforms the contained value of the [`Err`] variant:
+//! These methods transform the contained value of the [`Err`] variant:
 //!
 //! * [`map_err`] transforms [`Result<T, E>`] into [`Result<T, F>`] by
 //!   applying the provided function to the contained value of [`Err`] and

